pgsidecar/docker/Dockerfile

```diff
@@ -0,0 +1,24 @@
+FROM postgres:11 as base
+
+ARG github_user
+ARG github_token
+
+ENV GITHUB_USER ${github_user}
+ENV GITHUB_TOKEN ${github_token}
+
+RUN apt-get update \
+  && apt-get install -y wget \
+  && apt-get install -y postgresql-11-postgis-scripts
+
+RUN set -e && \
+  wget https://github.com/oneconcern/datamon/releases/download/v0.7/datamon.linux.tgz &&\
```

Comment:
in order to avoid cyclical dependencies of a sort, build datamon using alpine and then copy it into the postgres image as in `sidecar.Dockerfile` or use the existing `gcr.io/onec-co/datamon-fuse-sidecar` image as a place to copy the binary from.  write a makefile target to tie the entire build together.

one way or another, in-repo dockerfiles needn't cyclically depend on github releases that are themselves created out of the repo.